sd_dynamic_prompts/ui/prompt_writer.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+import csv
+from pathlib import Path
+
+from dynamicprompts import constants
+
+
+class PromptWriter:
+    def __init__(self):
+        self.reset()
+        self._enabled = False
+
+    def reset(self):
+        self._already_saved = False
+        self._positive_prompt = ""
+        self._negative_prompt = ""
+        self._positive_prompts = []
+        self._negative_prompts = []
```

Comment:
Maybe these should be called `original_..._prompt` and `generated_..._prompts` respectively?